webpack.dev.js

```diff
@@ -45,18 +44,19 @@ module.exports = merge.smart(require('./webpack.base.js'), {
       },
     }),
     new webpack.DefinePlugin({
-      NETLIFY_CMS_VERSION: JSON.stringify(require("./package.json").version + "-dev")
+      NETLIFY_CMS_VERSION: JSON.stringify(require("./package.json").version + "-dev"),
     }),
-    new webpack.optimize.OccurenceOrderPlugin(),
     new webpack.HotModuleReplacementPlugin(),
-    new webpack.NoErrorsPlugin(),
-    new ExtractTextPlugin('[name].css', { disable: true }),
+    new webpack.NoEmitOnErrorsPlugin(),
+    new ExtractTextPlugin({
+      filename: '[name].css',
+      disable: true,
+    }),
   ],
   devServer: {
     hot: true,
     contentBase: 'example/',
     historyApiFallback: true,
-    devTool: 'cheap-module-source-map',
```

Comment:
The sourcemaps are still there as far as I can tell, but I didn't yet find a documented place to move this setting.